domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -180,5 +200,5 @@ def log_error(self, error):
                       f"pending_claim_id={self.job.pending_claim_id} "
                       f"ep400_claim_id={self.job.ep400_claim_id} "
                       f"state={self.job.state} "
-                      f"error=\'{error}\'")
-        self.job.error(self.job.state, error)
+                      f"error=\"{jsonable_encoder(error)}\"")
+        self.job.error(error if type(error) is list else [error])
```

Comment:
Prefer `isinstance(error, list)` over `type(error) is list` to support subclasses.